package/src/lib/SpotlightTour.provider.tsx

```diff
@@ -87,125 +100,143 @@ export interface SpotlightTourProviderProps {
   steps: TourStep[];
 }
 
+const DEFAULT_FLOATING_CONFIGURATIONS: FloatingProps = {
+  middleware: [flip(), offset(4), shift()],
+  placement: "bottom",
+};
+
 /**
  * React provider component to get access to the SpotlightTour context.
  */
-export const SpotlightTourProvider = forwardRef<SpotlightTour, SpotlightTourProviderProps>((props, ref) => {
```

Comment:
There's a big reformat we don't need in this component 😓 let's revert it also to make the changes easy to go through.

Reply:
already fixed 👍